celestia/gentx/ActiveNodes.json

```diff
@@ -0,0 +1,62 @@
+{
+  "body": {
+    "messages": [
+      {
+        "@type": "/cosmos.staking.v1beta1.MsgCreateValidator",
+        "description": {
+          "moniker": "Active Nodes",
+          "identity": "A2EA7DE76AD57E1A",
+          "website": "https://www.activenodes.io",
+          "security_contact": "",
+          "details": "We Skyrocket your Stake! Highly Available Enterprise Infrastructure with multiple nodes around the world"
+        },
+        "commission": {
+          "rate": "0.050000000000000000",
+          "max_rate": "0.200000000000000000",
+          "max_change_rate": "0.050000000000000000"
+        },
+        "min_self_delegation": "1",
+        "delegator_address": "celestia1t345w0vxnyyrf4eh43lpd3jl7z378rtsgv8j5h",
+        "validator_address": "celestiavaloper1t345w0vxnyyrf4eh43lpd3jl7z378rtsdn9tz3",
+        "pubkey": {
+          "@type": "/cosmos.crypto.ed25519.PubKey",
+          "key": "J+TjVEWF8QcOdjJAfuaVjK+AlZdUn3wK5qW5DrVIDFo="
+        },
+        "value": {
+          "denom": "utia",
+          "amount": "72850000000"
+        }
+      }
+    ],
+    "memo": "ffffffffffffffffffffffffffffffffffffffff@255.255.255.255:26656",
```

Comment:
Can we make the memo empty instead?

Reply:
It's not possible to keep memo empty, so this my default from offline gentx node.
What's the problem? as you can see it works perfectly in other chains. Here for example kujira mainnet genesis.json: https://raw.githubusercontent.com/Team-Kujira/networks/master/mainnet/kaiyo-1.json